### ISSUE TITLE
Is the "humhub.enterprise.theme.js" working?

### ISSUE THREAD

**Issue by tomofu74** [humhub/humhub#3845]
If I edited that file, but it does not effective.
the file is placed 'themes/"my-custom-theme-name"/js' folder.

### What steps will reproduce the problem?
 In the debugger, like CHROME, debugger console sources -> design
 that js file does not seem to be listed...

### What is the expected result?
 I want to make a custom JS run.

### What do you get instead?
 No effects. (No error displayed)

### Additional info

| Q                | A
| ---------------- | ---
| HumHub version   | 1.3.17 Enterprise edition.
| PHP version      | 7.2.21
| Operating system | Windows Server 2016

> Note: Also provide the full error stacks from `Administration --> Information --> Logging` and errors from your browser console if possible.
> Note: Please use the [community](https://community.humhub.com) site for any questions on customizing HumHub.

**Comment by luke-** [comment]
Please use our community for questions around theming.
You may need to overwrite also a header view file to register your javascript.

**Comment by tomofu74** [comment]
Thank you, luke.
I'll try to overwrite a header view file.

**Comment by buddh4** [comment]
The enterprise module will always include the root script file. What do you whish to change? Also note that there is a [repository](https://github.com/humhub/enterprise) for enterprise related issues.

**Comment by tomofu74** [comment]
Hi, buddh4. Thank you for your interest. We wish to fix the scrolling bug which is noted on enterprise repository related issues the same as [this](https://github.com/humhub/enterprise/issues/7). We look forward to fixing them. We hope for your success.

**Comment by buddh4** [comment]
I tried to fix it in between some other issues, but did not manage to stop horizontal scrolling on mobile. I'll have another try as early as I can. If you want to try to fix it you'll have to add the following to `EnterpriseThemeAsset`:

```
 public $publishOptions = [
        'forceCopy' => true
    ];

```

This will republish the `humhub.enterprise.theme.js` on every request. Don't forget to set it to false when you've stopped working on it.

**Comment by buddh4** [comment]
I've fixed the issue. See https://github.com/humhub/enterprise/issues/7

A total fix will require ee theme update + HumHub 1.4.2. Both updates will be available soon.
